src/nm-connectivity.c

```diff
@@ -41,7 +41,7 @@ typedef struct {
 	struct curl_slist *request_headers;
 	guint timeout_id;
 	char *ifspec;
-} ConCheckCbData;
+} NMConnectivityCheckHandle;
```

Comment:
Since the structure is local, can we avoid to namespace it with the NM
prefix? For example, "ConnectivityCheckHandle"

Reply:
This is a trival commit that does only renaming in preparation of the follow up commits. The follow up commit makes this handle public (at least, it's name. It will still be an opaque structure).